internal/core/data/v2/controller/http/event_test.go

```diff
@@ -751,3 +749,74 @@ func TestAllEventsByTimeRange(t *testing.T) {
 		})
 	}
 }
+
+func TestAllReadings(t *testing.T) {
+	testLabels := []string{"label1", "label2"}
+
+	dic := mocks.NewMockDIC()
+	dbClientMock := &dbMock.DBClient{}
+	dbClientMock.On("AllReadings", 0, 20, []string(nil)).Return([]models.Reading{}, nil)
+	dbClientMock.On("AllReadings", 0, 1, testLabels).Return([]models.Reading{}, nil)
+	dic.Update(di.ServiceConstructorMap{
+		v2DataContainer.DBClientInterfaceName: func(get di.Get) interface{} {
+			return dbClientMock
+		},
+	})
+	controller := NewEventController(dic)
+	assert.NotNil(t, controller)
+
+	tests := []struct {
+		name               string
+		offset             string
+		limit              string
+		labels             string
+		errorExpected      bool
+		expectedStatusCode int
+	}{
+		{"Valid - get readings without offset, limit, and labels", "", "", "", false, http.StatusOK},
```

Comment:
Should have a case that returns a couple readings and verify expected count.

Reply:
Yeah, I thought this case should be part of reading_test in application layer, so added the case over [here](https://github.com/edgexfoundry/edgex-go/pull/2897/files#diff-b9258d37854fc98a7cb4c09ad4291b0a6deb205a1245107ac78370a8c9f7f325R39).